doc/config_rsmtool.rst

```diff
@@ -165,6 +165,8 @@ RSMTool provides pre-defined sections for ``rsmtool`` (listed below) and, by def
         - the variances; and
         - a Scree plot.
 
+    The analysis keeps all components. The total number of components usually equals the total number of features. In cases where the total number of responses is smaller than the number of features, the number of components is the same as the number of responses.  
```

Comment:
This paragraph needs to be indented two spaces beyond where the `pca` bullet starts.